lineman/app/models/notification_records_interface.coffee

```diff
@@ -2,6 +2,13 @@ angular.module('loomioApp').factory 'NotificationRecordsInterface', (BaseRecords
   class NotificationRecordsInterface extends BaseRecordsInterface
     model: NotificationModel
 
+    fetchMyNotifications: ->
+      @fetch
+        params:
+          from: 0
```

Comment:
nope.. paging does not makes sense in this context. We can only grab most
recent records with this..

On Wed, 13 May 2015 at 03:13 James Kiesel <notifications@github.com> wrote:

> In lineman/app/models/notification_records_interface.coffee
> <https://github.com/loomio/loomio/pull/2149#discussion_r30144210>:
>
> > @@ -2,6 +2,13 @@ angular.module('loomioApp').factory 'NotificationRecordsInterface', (BaseRecords
> >    class NotificationRecordsInterface extends BaseRecordsInterface
> >      model: NotificationModel
> >
> > +    fetchMyNotifications: ->
> > +      @fetch
> > +        params:
> > +          from: 0
>
> It might do to have a chat about paging with this system, which will be a
> bit wonky; I don't think we want to be combining timeframe and page too
> much. (Give me the 25th - 50th record created in the last day? Blerg. Even
> 'give me the last 20 records created in the last timeframe' doesn't work,
> because what if someone comes back after an hour and 30 things have been
> created? Still have a hole.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2149/files#r30144210>.
>